docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,247 @@ sidebar_position: 2
+kind: Deployment    ############################### RESOURCE
+
+metadata:
+  name: nodejs-app  #################### NAME
+  labels:
+    app: nodejs
+    runtime: nodejs
+    runtime-version: 14-ubi7
+
+spec:
+  selector:
+    matchLabels:
+      app: nodejs
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nodejs
+        deploymentconfig: nodejs
+    spec:
+      containers:
+      - name: nodejs
+        image: nodejs:latest
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+EOD
+```
+
+Application can be a podSpec based resource like "Deployment" or "DeploymentConfig" or a non podSepc based resource like a Custom Resource Definition.
+
+### Service
+
+Apply the following CatalogSource:
+
+```yaml
+kubectl apply -f - << EOD
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+    name: sample-db-operators
+    namespace: openshift-marketplace
+spec:
+    sourceType: grpc
+    image: quay.io/redhat-developer/sample-db-operators-olm:v1
+    displayName: Sample DB Operators
+EOD
+```
+
+Apply the following Subscription:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: db-operators
+    namespace: openshift-operators
+spec:
+    channel: stable
+    installPlanApproval: Automatic
+    name: db-operators
+    source: sample-db-operators
+    sourceNamespace: openshift-marketplace
+    startingCSV: postgresql-operator.v0.0.8
+```
+
+Create a postgresql database:
+
+```yaml
+kubectl apply -f - << EOD
+---
+apiVersion: postgresql.baiju.dev/v1alpha1 ########### GROUP/VERSION
+kind: Database ###################################### RESOURCE
+metadata:
+  name: db-demo ##################################### NAME
+  namespace: service-binding-demo
+spec:
+  image: docker.io/postgres
+  imageName: postgres
+  dbName: db-demo
+EOD
+```
+
+### Service Binding
+
+To connect the `nodejs-app` Deployment with `db-demo` Database,
+create a `ServiceBinding` custom resource which includes both Deployment and Database metadata:
+- Group 
+- Version 
+- Resource 
+- Name
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+
+metadata:
+  name: binding-request 
+
+spec:
+
+  application:
+    group: apps 
+    version: v1
+    resource: deployments
+    name: nodejs-app
+
+  service:
+   group: postgresql.baiju.dev
+   version: v1alpha1
+   resource: Database
+   name: db-demo
+```
 
-_Here we provided a small sample on K8s_
+The Service Binding status should be updated:
+
+```yaml
+status:
+  conditions:
+  - lastHeartbeatTime: "2021-05-17T09:05:46Z"
+    lastTransitionTime: "2021-05-17T07:09:48Z"
+    status: "True"
+    type: CollectionReady
+  - lastHeartbeatTime: "2021-05-17T09:05:46Z"
+    lastTransitionTime: "2021-05-17T07:09:48Z"
+    status: "True"
+    type: InjectionReady
+  - lastHeartbeatTime: "2021-05-17T09:05:46Z"
+    lastTransitionTime: "2021-05-17T07:09:48Z"
+    status: "True"
+    type: Ready
+```
```

Comment:
but the major thing we do not speak about that the application now works, and it did not before applying the binding